src/dialogs/notifications/notification-drawer.js

```diff
@@ -25,7 +25,7 @@ export class HuiNotificationDrawer extends EventsMixin(
         color: var(--primary-text-color);
         border-bottom: 1px solid var(--divider-color);
         background-color: var(--primary-background-color);
-        min-height: 64px;
+        height: calc(var(--header-height) - 1px);
```

Comment:
```suggestion
        height: var(--header-height);
        box-sizing: border-box;
```
?

Reply:
border-box doesn't work in this particular situation, unless we adjust some other styling as well. With your changes above:

![image](https://user-images.githubusercontent.com/25127328/97183723-7fa38480-1774-11eb-8e2d-77901b2738e0.png)

Reply:
We could use box-sizing if we remove the width calculation on the same element `width: calc(100% - 32px);`